spec/github_url_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+describe 'URL Building' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.